kitsune/users/views.py

```diff
@@ -330,6 +330,11 @@ def get_settings(attr, *args):
     def get(self, request):
         is_contributor = request.GET.get("is_contributor") == "True"
         request.session["is_contributor"] = is_contributor
+
+        locale = getattr(request, "LANGUAGE_CODE", None)
```

Comment:
how about request.get('LANGUAGE_CODE', '')?

Reply:
or maybe something like request.get('LANGUAGE_CODE', 'en-US') and set it directly to the session